atomate2/openmm/jobs/base.py

```diff
@@ -0,0 +1,46 @@
+
+from typing import Callable
+from jobflow import job, Maker
+from dataclasses import dataclass
+from pathlib import Path
+from pymatgen.io.openmm.sets import OpenMMSet
+from atomate2.openmm.schemas.task import OpenMMTaskDocument
+
+def openmm_job(method: Callable):
+    """
+
+    Parameters
+    ----------
+    method : callable
+        A BaseVaspMaker.make method. This should not be specified directly and is
```

Comment:
```suggestion
        A BaseOpenmmMaker.make method. This should not be specified directly and is
```

This looks like a copy pasta typo

Reply:
Yep, probably more where that came from lol